src/YoutubeAtom.tsx

```diff
@@ -160,6 +160,7 @@ export const YoutubeAtom = ({
                         role={role}
                         duration={duration}
                         pillar={pillar}
+                        title={title}
```

Comment:
Do we also need to update `YoutubeAtom.test.tsx`?

Reply:
Added tests for `aria-label` and `title`